README.md

```diff
@@ -0,0 +1,41 @@
+<h1 align="center">Games 🎲 (BETA) </h1>
+
+<h2 align="center" >
+    Some simples games for fun and Study Python 🧩 🎮 <br>
+    <img alt="GitHub stars" src="https://img.shields.io/github/stars/YagoMaia/Games?style=social">
+    <img alt="GitHub followers" src="https://img.shields.io/github/followers/YagoMaia?label=Follow%20me%20%3A%29&style=social">
+</h2>
+<p align="center">
+    <img alt="Games" src="">
+</p>
+
+## 💭 Description:
+
+<p>
+   Three simples tests games in beta to study
+</p>
+
+## 🕹️ Usage:
```

Comment:
Aqui vc pode descrever como funciona o projeto, os testes aí, o X1 (Se vc não for colocar ele em um repositório separado) principalmente, e o jogo de escolhas, vou criar um desse tbm, mas estilo Undertale, seu jogo me inspirou muito man, vlw mesmo